### DISCUSSION TITLE
CUANTOS PROYECTOS SON FRONT???

### DISCUSSION THREAD

**Discussion by nathalieCarrasco** [Launch-X-Latam/MisionFrontEnd#431] [General]
hola si alguno tuviera el total de proyectos necesito ponerme al corriente se agradece

**Comment by JuanSposada** [comment]
https://github.com/Launch-X-Latam/MisionFrontEnd

en cada una de las carpetas que vienen en el repositorio, viene una que dice practicas, en algunas unidades solo es una, en otras son mas, debes cehcarlo, saludos
